stdlib.sh

```diff
@@ -815,6 +815,21 @@ layout_php() {
   PATH_add vendor/bin
 }
 
+# Usage: layout pixi [args]
+#
+# Loads a pixi environmnt.
+# If no additional arguments are given the `default` environment is loaded.
+# You can pass `-e <env_name>` to load a different environment instead.
+# For supported arguments see `pixi shell-hook --help`.
+layout_pixi() {
+    if [[ ! -f "pixi.toml" ]]; then
```

Comment:
Since we also support `pyproject.toml`, would it be great to take that case into account as well 🙂